libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/lib/messages.ts

```diff
@@ -42,6 +42,67 @@ export const medicalAndRehabilitationPaymentsFormMessage: MessageDir = {
 
   pre: defineMessages({
     // Data collection
+    skraInformationTitle: {
+      id: 'marp.application:pre.skra.information.title',
+      defaultMessage: 'Upplýsingar frá Þjóðskrá, RSK og Útlendingarstofnun',
+      description:
+        'Information from National Registry, RSK and Directorate of Immigration',
```

Comment:
Update translation? So it matches core translation?
<img width="349" alt="image" src="https://github.com/user-attachments/assets/ec870280-f468-4cd5-ad52-68b990fdfae4" />

```suggestion
        'Information from Registers Iceland, RSK and Directorate of Immigration',
```

Reply:
Description updated